graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/messages/RemoveFromStream.java

```diff
@@ -99,7 +100,11 @@ public FunctionDescriptor<Void> descriptor() {
                         nameParam,
                         idParam,
                         messageParam))
-                .description("Removes a message from a stream. Removing the last stream will put the message back onto the default stream. To complete drop a message use the drop_message function.")
+                .description("Removes a message from a stream. Removing the last stream will put the message back onto the default stream. To complete drop a message use the drop_message function. If no specific message is provided, it removes the message from the currently processed message.")
```

Comment:
"To complete**ly** drop"

I don't understand the last sentence:  "If no specific message is provided, it removes the message from the currently processed message."